packages/ui/src/ui/pages/system/Masters/SwitchLeader.tsx

```diff
@@ -0,0 +1,145 @@
+import React, {useCallback, useState} from 'react';
+import Icon from '../../../components/Icon/Icon';
+import Button from '../../../components/Button/Button';
+import ClipboardButton from '../../../components/ClipboardButton/ClipboardButton';
+import block from 'bem-cn-lite';
+import {YTApiId, ytApiV4Id} from '../../../rum/rum-wrap-api';
+import {wrapApiPromiseByToaster} from '../../../utils/utils';
+import './SwitchLeader.scss';
+import {YTDFDialog} from '../../../components/Dialog/Dialog';
+import {SwitchLeaderShortInfo} from '../../../pages/components/SwitchLeaderShortInfo/SwitchLeaderShortInfo';
+import {AppStoreProvider} from '../../../containers/App/AppStoreProvider';
+
+const b = block('switch-leader-button');
+
+type SwitchLeaderPromtProps = {
+    cancel: () => void;
+    confirm: () => void;
+    visible: boolean;
+    currentLeaderAddress: string;
+    newLeaderAddress: string;
+};
+
+const SwitchLeaderPromt = (props: SwitchLeaderPromtProps) => {
```

Comment:
We prefer `Dialog`/`Modal` suffix for such components. It is not an issue, but can you please rename it to `SwitchLeaderDialog`/`SwitchLeaderModal`.